webroot/results/includes/regions_history.php

```diff
@@ -26,7 +26,7 @@ function showRegionalRecordsHistory () {
   #--- Order: normal history or mixed?
   $order = $chosenHistory
            ? 'event.rank, type, value, year desc, month desc, day desc, roundType.rank desc'
-           : 'year desc, month desc, day desc, roundType.rank desc, event.rank, type, value';
+           : 'year desc, month desc, day desc, event.rank, type, value, roundType.rank desc';
```

Comment:
Does this change other orderings at all? I haven't looked through the git history at all, but I wonder if the reason for ranking by `roundType.rank` before `value` was for average results? Like if we changed feet from mean of 3 to average of 5, we might want things to show up in a particular order?

Reply:
It does change something in some case. I think this order should be fine. Both Mo3 and Ao5's type are `average` so it doesn't matter here.